src/get_ensemble_output.R

```diff
@@ -0,0 +1,43 @@
+library(hubUtils)
+library(hubVis)
+library(hubEnsembles)
+library(hubData)
+library(dplyr)
+
+
+hub_path <- "/Users/dk29776/Dropbox/UTAustin/flu-metrocast"
+hub_con <- connect_hub(hub_path)
+
+model_names <- hub_con %>%  
+  select(model_id) %>% 
+  distinct() %>% 
+  collect()
+
+model_names
+
+reference_date = "2025-03-01"
+
+
+linear_pool_norm <- hub_con %>%
+  filter(
+    reference_date == .env$reference_date,
+    !(model_id %in% c("epiENGAGE-ensemble_mean", "epiENGAGE-lop_norm"))
+  ) %>%
+  collect_hub() %>%
+  hubEnsembles::linear_pool(model_id = "linear-pool-normal")|>
+  select(-model_id)
+dim(linear_pool_norm)
+write.csv(linear_pool_norm, paste("model-output/epiENGAGE-lop_norm/",reference_date, "-epiENGAGE-lop_norm.csv", sep = ""), row.names = FALSE)
+
+
+mean_ens <- hub_con %>%
+  filter(
+    reference_date == .env$reference_date,
+    !(model_id %in% c("epiENGAGE-ensemble_mean", "epiENGAGE-lop_norm"))
+  ) %>%
+  collect_hub() %>%
```

Comment:
minor point, but it might make more sense to only do the `collect_hub()` call once, so you only load the data once and then run the different ensemble algorithms on the same data object. Minor point, in practice won't save much time, and might be more important if at some point we moved the data to the cloud and accessed/downloaded it from there.

I guess the larger argument for doing it this way is that if you ever change the logic about which data to pull down, you would only change it in one place instead of two.